packages/frontend-2/components/connectors/Card.vue

```diff
@@ -4,12 +4,19 @@
       class="flex flex-1 flex-col gap-1 !p-4 !pt-2 !pb-3 h-full hover:bg-foundation"
     >
       <div class="flex gap-2 items-center">
-        <img
-          v-if="connector.image"
-          :src="connector.image"
-          :alt="`${connector.title} logo`"
-          class="w-[48px] -ml-1"
-        />
+        <div v-if="connector.images" class="relative flex items-start mr-2">
+          <div
+            v-for="(image, index) in connector.images"
+            :key="image"
+            :class="[
```

Comment:
We can probably improve this, but somebody needs to improve these icons

Reply:
yup, you are the boss